src/core/unittest/FrameTest.cpp

```diff
@@ -243,6 +243,43 @@ TEST(FrameTest, ReliableResetStreamFrameEncodeDecode)
     ASSERT_EQ(Frame.ReliableSize, DecodedFrame.ReliableSize);
 }
 
+TEST(FrameTest, TestQuicAckTrackerDidHitReorderingThreshold)
```

Comment:
https://datatracker.ietf.org/doc/html/draft-ietf-quic-ack-frequency#section-6.2.1 actually has a lot of examples. Could we put all these into a unit test?